src/Neo.CLI/Dockerfile

```diff
@@ -1,20 +1,43 @@
-FROM --platform=$BUILDPLATFORM mcr.microsoft.com/dotnet/sdk:9.0 AS Build
+FROM mcr.microsoft.com/dotnet/aspnet:9.0-noble
 
-# Run this from the repository root folder
-COPY src . 
-COPY NuGet.Config /Neo.CLI
+# Install all dependencies in a single RUN to reduce layers and speed up build
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        net-tools telnet bash-completion wget lrzsz zip \
+        software-properties-common \
+        apt-transport-https \
+        build-essential \
+        unzip \
```

Comment:
tmux?

Reply:
What's tmux?

Reply:
An awesome tool, but don't care, ready to me